unlock_indicator.c

```diff
@@ -21,8 +21,8 @@
 #include "xinerama.h"
 
 #define BUTTON_RADIUS 90
-#define BUTTON_SPACE (BUTTON_RADIUS + 5)
-#define BUTTON_CENTER (BUTTON_RADIUS + 5)
+#define BUTTON_SPACE (BUTTON_RADIUS + 20)
```

Comment:
I’m not convinced these visual changes are clear enough/make sense. Can you attach a screenshot of how this looks like with/without caps lock?